std_msgs/Header header
int32 id
float64 size
geometry_msgs/Pose pose
